mopidy_pandora/library.py

```diff
@@ -161,11 +154,12 @@ def get_images(self, uris):
                     )
                 pass
             result[uri] = [models.Image(uri=u) for u in image_uris]
+        print(f"\n\n\nIMAGES: {result}\n\n\n")
         return result
 
     def _formatted_station_list(self, list):
         # Find QuickMix stations and move QuickMix to top
-        for i, station in enumerate(list[:]):
+        for i, station in enumerate(list):
```

Comment:
It looks like the original code relied on making a copy of the list so that we can modify the original while iterating over the values. Wont' this change cause an exception on line 167 further below?

Reply:
Okay, so this is one of the areas where my limited knowledge of Python left me wondering.  I didn't realize that the [:] syntax made a copy of the list, bu with [:] the enumerate failed with a message about the type not being a list.  I'm not getting any exceptions when I run the test, but I'm not sure if line 167 is covered.  Did python 3 change the [:] syntax?

Reply:
It could be that the enumerate is failing because we are are using the reserved keyword `list` as the variable name? 

I was learning Python myself when this was written 5 or so years ago, and that variable name is obviously problematic in retrospect.

Reply:
I updated this to be enumerate(station_list.copy()).  Python complained with `TypeError: unhashable type: 'slice'` when using the [:] copy syntax, but .copy() seems to work fine.

Reply:
Using the syntax `foo[:]` is the same as calling `foo.__getattr__(slice(0, len(foo))`, and that should not have changed in Python 3.

I suspect that `list` isn't a regular list, but some other object implementing `__getattr__()`, without adding explicit support for slices in the `__getattr__()` implementation.

Using `.copy()` sounds like a good solution.